include/hashinator/hashinator.h

```diff
@@ -56,20 +56,30 @@ using DefaultMetaAllocator = split::split_host_allocator<T>;
 #endif
 
 using MapInfo = Hashinator::Info;
-template <typename KEY_TYPE, typename VAL_TYPE, KEY_TYPE EMPTYBUCKET = std::numeric_limits<KEY_TYPE>::max(),
-          KEY_TYPE TOMBSTONE = EMPTYBUCKET - 1, class HashFunction = HashFunctions::Fibonacci<KEY_TYPE>,
-          class DeviceHasher = DefaultHasher, class Meta_Allocator = DefaultMetaAllocator<MapInfo>>
+template <typename KEY_TYPE, typename VAL_TYPE, class Allocator = DefaultMetaAllocator<hash_pair<KEY_TYPE,VAL_TYPE>>,
```

Comment:
Why is the default called a DefaultMetaAllocator? (emphasis on Meta)

Reply:
No reason so this is now updated